src/ploomber/tasks/notebook.py

```diff
@@ -599,6 +614,8 @@ def __init__(
         check_if_kernel_installed=True,
         debug_mode=None,
     ):
+        self.engine = engine
```

Comment:
let's validate that the value is either ploomber-engine or papermill, raise a ValueError if it's a different value and show the valid values